src/util/map.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+
+/**
+ * Convert a string-keyed map to an object. Useful for conversion to
+ * JSON. If a map's keys are not strings, consider invoking `mapKeys`
+ * first.
+ */
+export function toObject<K: string, V, InK: K, InV: V>(
```

Comment:
I find the type signature to this function confusing. Could the docstring explain it, please?